dev-docs/bidders/insticator.md

```diff
@@ -6,7 +6,7 @@ biddercode: insticator
 gdpr_supported: true
 usp_supported: true
 schain_supported: true
-media_types: banner
+media_types: banner, video
 pbjs: true
 gvl_id: 910
```

Comment:
Thanks for updating the docs. Could you add the flag multiformat_supported (introduced in https://github.com/prebid/prebid.github.io/pull/3694 ) as well?

It allows sales houses / publishers to better understand the behaviour of the bid adapter for multi format ad units.

```
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
```